src/mixins/java/org/spongepowered/common/mixin/tracker/world/chunk/ChunkMixin_OwnershipTracked.java

```diff
@@ -88,7 +88,9 @@
             + "/function/Consumer;)V", at = @At("RETURN"))
     private void tracker$setUpUserService(final World world, final ChunkPos pos, final Biome[] biomes, final UpgradeData data, final ITickList<Block> blockTickList,
                                           final ITickList<Fluid> fluidTickList, final long inhabitedTime, final ChunkSection[] sections, final Consumer<Chunk> chunkConsumer, final CallbackInfo ci) {
-        this.tracker$userStorageService = world != null && !((WorldBridge) world).bridge$isFake() ? null : Sponge.getServiceProvider().userStorageService();
+        this.tracker$userStorageService = world != null && !((WorldBridge) world).bridge$isFake()
+                                  ? null
+                                  : SpongeCommon.getGame().getServiceProvider().userStorageService();
```

Comment:
This was causing a big performance issue in api7.

It should use the user storage service when the world is **not** fake (not the opposite).
(see `ChunkMixin_Tracker`)

Reply:
I'm not interested in making logic changes outside the replacement of the service system in this PR because otherwise it'll never end.